server/app/services/program/ProgramService.java

```diff
@@ -781,10 +817,34 @@ public ErrorAnd<ProgramDefinition, CiviFormError> updateLocalization(
     ProgramDefinition programDefinition = getFullProgramDefinition(programId);
     ImmutableSet.Builder<CiviFormError> errorsBuilder = ImmutableSet.builder();
     validateProgramText(errorsBuilder, "display name", localizationUpdate.localizedDisplayName());
-    validateProgramText(
-        errorsBuilder, "short display description", localizationUpdate.localizedShortDescription());
+
+    // only validate the short description if the program has one
+    // short description is now a required field, but older programs might not have one yet
+    Optional<String> shortDescription =
+        programDefinition.localizedShortDescription().maybeGet(LocalizedStrings.DEFAULT_LOCALE);
+    if (shortDescription.isPresent() && !shortDescription.get().isBlank()) {
+      validateProgramText(
+          errorsBuilder,
+          "short display description",
```

Comment:
Step 1 is the only application step that's required and we generally validate translations the same way we do for program fields, so only error on the required ones. That said, it would be a little weird if there's only a translation for one step but not the others... here's a slack convo with more discussion: https://exygy.slack.com/archives/C06NZ470LAU/p1734456551876609 What do you think?

Reply:
That's a good call-out that the application step validation probably needs a comment though. I'll add one.

Reply:
Yeah that would be a funky user experience. If you're ok with it, I'm inclined to merge this as is since this is what product/design asked for and then reopen that conversation and fix it in january. It's not live/used anywhere yet so we've got time.